webpack/react_app/components/JobWizard/steps/CategoryAndTemplate/CategoryAndTemplate.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Title } from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { SelectField } from '../form/SelectField';
+import { GroupedSelectField } from '../form/GroupedSelectField';
+
+export const CategoryAndTemplate = ({
+  jobCategories,
+  jobTemplates,
+  setJobTemplate,
+  selectedTemplateID,
+  selectedCategory,
+  setCategory,
+}) => {
+  const templatesGroups = {};
+  jobTemplates.forEach(template => {
+    if (templatesGroups[template.provider_type])
+      templatesGroups[template.provider_type].options.push({
+        label: template.name,
+        value: template.id,
+      });
+    else
+      templatesGroups[template.provider_type] = {
+        options: [{ label: template.name, value: template.id }],
+        groupLabel: template.provider_type,
+      };
+  });
+
+  const selectedTemplate = jobTemplates.find(
+    template => template.id === selectedTemplateID
+  )?.name;
+
+  const onSelectCategory = newCategory => {
+    setCategory(newCategory);
+    setJobTemplate(null);
+  };
+  return (
+    <>
```

Comment:
An empty tag?

Reply:
https://reactjs.org/docs/fragments.html#short-syntax
Reacts asks that the `return` only returns one element so I'm wrapping all the elements

Reply:
Oh, wow. I was aware of `React.Fragment`, but not the short syntax. Nevermind then